tracer/src/Datadog.Trace/RemoteConfigurationManagement/Protocol/GetRcmRequest.cs

```diff
@@ -21,5 +23,12 @@ public GetRcmRequest(RcmClient client, List<RcmCachedTargetFile> cachedTargetFil
 
         [JsonProperty("cached_target_files")]
         public List<RcmCachedTargetFile> CachedTargetFiles { get; }
+
+        public bool Matches(GetRcmResponse response)
```

Comment:
This is only used for testing, right? Maybe should be moved into the test folder? Or perhaps given a clear name, I find it hard to understand what `Match` means without other context.

Reply:
good point thanks!!

Reply:
fixed in aa4464a434a87f979e22ec38b29edacee4ec4475